script/configgen.sh

```diff
@@ -16,34 +16,62 @@ if [[ $(uname) == "Darwin" ]]; then
 	SED_EW='[[:>:]]' #End of word in regex
 fi
 
+# ifd-from-ansible is responsible for generating the 'infrastructure-data' that is
+# read by the testnet generator tool.
+ifd-from-ansible() {
+	HOST_PATH=$1
+	OUT_PATH=$2
+	
+	cat <<EOF > $OUT_PATH
+	{
+	  "provider": "digital-ocean",
+	  "network": "0.0.0.0/0",
+	  "instances": {
+EOF
+	
+	lines=`grep '^.*name=.*$' $HOST_PATH | tr " " : `;
```

Comment:
[for my own instruction] I don't get the semicolon

Reply:
Ah, not needed. Leftover from a previous way of doing this.